src/insights_client/__init__.py

```diff
@@ -362,21 +413,27 @@ def _main():
         valid_env_egg = []
 
     if not validated_eggs and not valid_env_egg:
-        sys.exit("No GPG-verified eggs can be found")
+        sys.exit("No GPG-verified initial eggs can be found")
 
     # ENV egg comes first
-    sys.path = valid_env_egg + validated_eggs + sys.path
+    all_valid_eggs = valid_env_egg + validated_eggs
+    logger.debug("Using eggs: %s", join_path(all_valid_eggs))
+    sys.path = all_valid_eggs + sys.path
 
     try:
+        logger.debug("Importing the insights module from %s...", join_path(sys.path))
```

Comment:
Information on this line is completely covered by the `Using eggs:` log message:
```
2024-05-15 11:19:10,159    DEBUG __main__:420 Using eggs: /.../insights-core:/var/lib/insights/last_stable.egg
2024-05-15 11:19:10,160    DEBUG __main__:424 Importing the insights module from /.../insights-core:/var/lib/insights/last_stable.egg:/.../insights-client/src/insights_client:/.../insights-client/src:/usr/lib64/python312.zip:/usr/lib64/python3.12:/usr/lib64/python3.12/lib-dynload:/usr/lib64/python3.12/site-packages:/usr/lib/python3.12/site-packages...
```

Reply:
Removed.